lib/console.js

```diff
@@ -166,6 +170,36 @@ Console.prototype.assert = function assert(expression, ...args) {
   }
 };
 
+// Defined by: https://console.spec.whatwg.org/#clear
+Console.prototype.clear = function clear() {
+  // It only makes sense to clear if _stdout is a TTY.
+  // Otherwise, do nothing.
+  if (this._stdout.isTTY) {
+    const { cursorTo, clearScreenDown } = require('readline');
+    cursorTo(this._stdout, 0, 0);
+    clearScreenDown(this._stdout);
+  }
+};
+
+// Defined by: https://console.spec.whatwg.org/#count
+Console.prototype.count = function count(label = kDefaultCount) {
```

Comment:
According to the [IDL](https://console.spec.whatwg.org/#console-namespace), `label` is a DOMString with default being `'default'`. This means that the implementation should be akin to:

```js
function count(label = 'default') {
  label = `${label}`;
  // ...
  this.log(`${label}: ${counts[label]}`);
}
```

This should also allow the use of `Map` for `this[kCounts]`.